parser-wrapper/interface-impl-v1/src/main/java/org/mule/raml/implv1/model/parameter/ParameterImpl.java

```diff
@@ -88,5 +84,15 @@ public boolean isStringArray() {
         return false;
     }
 
+    @Override
+    public boolean isScalar() {
+        return true;
+    }
+
+    @Override
+    public boolean isFacetArray(String facet) {
+        return false;
+    }
+
```

Comment:
In RAML 0.8, are parameters always scalar types?

Reply:
In 0.8 the concept of array doesn't exist

Reply:
You are right. Dismiss the comment